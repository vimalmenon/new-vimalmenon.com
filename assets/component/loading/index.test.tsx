import React from "react";
import renderer from "react-test-renderer";

import Loading from "./index";

describe("Loading Component", ()=> {
    test("Snapshot Testing", () => {
        const component = renderer.create(<Loading />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});