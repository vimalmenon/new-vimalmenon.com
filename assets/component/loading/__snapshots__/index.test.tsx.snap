// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component Snapshot Testing 1`] = `
<section
  className="makeStyles-root-1"
>
  <div
    className="makeStyles-loadingWrapper-2"
  >
    <div
      className="makeStyles-loader-3"
    />
  </div>
</section>
`;
